packages/app-wallet-interface/source/index.html.md

```diff
@@ -410,6 +410,62 @@ Possible response to a `Channel Proposed` event.
 | ---- | ----------------- | ---------------------------------------------------------------- |
 |      | Channel not found | The wallet can't find the channel corresponding to the channelId |
 
+## Challenge Channel
+
+```json
+{
+  "jsonrpc": "2.0",
+  "method": "ChallengeChannel",
+  "id": 0,
+  "params": {
+    "channelId": "0xabc123"
+  }
+}
+```
+
+> Example response
+
+```json
+{
+  "jsonrpc": "2.0",
```

Comment:
Are we returning the `challengeTimeout`? We should let the app know when the challenge will expire.

Reply:
Just added that.